test/functional/autocmd/termxx_spec.lua

```diff
@@ -123,6 +123,7 @@ it('autocmd TermEnter, TermLeave', function()
   command('wincmd w')
   feed('i')
   command('q!')
+  helpers.retry_inf(function()
```

Comment:
+1, this is testing an asynchronous behavior. so this makes sense.
```suggestion
  retry(function()
```